src/lib/helpers/jsToAqua.ts

```diff
@@ -85,11 +109,11 @@ export const jsToAqua = (
   );
 
   return `${
-    typeof typeDefs === "string" ? `${typeDefs}\n\n` : ""
+    typeDefs === undefined ? "" : `${typeDefs}\n\n`
```

Comment:
What can `typeDefs` be `undefined`?

Reply:
these are typeDefs for objects. When it's a scalar - typeDefs are undefined - you check out tests to see that